ogc_ws/src/despatcher/src/gnd_despatcher.py

```diff
@@ -122,7 +126,25 @@ def check_incoming_msgs(self, data):
                     self.pub_to_rqt_ondemand.publish(data.data)
         except (ValueError, IndexError):
             rospy.logerr("Invalid message format!")
-    
+
+    # check whether multi aircraft works
+    def rqt_recovery_log(self, entries, aircraft_id):
+        rospy.logerr(entries)
+        path = os.path.join(rospkg.RosPack().get_path("yonah_rqt"), "src/yonah_rqt", "demofile.txt")
```

Comment:
Is demofile.txt the name of the recovery log? Is the file name static or can we change it to whatever name we want?

Reply:
I changed the system slightly from the previous one now that you mentioned this. Previously there needs to be a txt file named demofile.txt in order for it to work. Now, the rqt automatically checks whether the filename exists (I changed the filename to rqt_log.txt) and if it doesn't exist, it will create a new txt file. 

As for whether the name is static, it technically is assuming you are talking about changing the filename externally. The user can change the filename to whatever you want, but this must be done through the code, there are 2 places where this needs to be done, main_window.py and gnd_despatcher.py, there is a variable called filename